gateway/modules/eventing/operations.go

```diff
@@ -25,7 +25,7 @@ func (m *Module) IsEnabled() bool {
 }
 
 // QueueEvent queues a new event
-func (m *Module) QueueEvent(ctx context.Context, project, token string, req *model.QueueEventRequest) (interface{}, error) {
+func (m *Module) QueueEvent(ctx context.Context, project, eventDocID, batchID, token string, eventToken int, req *model.QueueEventRequest) (interface{}, error) {
```

Comment:
Don't change this function. Revert back to original for this one. Remove the tests for it too.